models/asymkey/ssh_key_deploy.go

```diff
@@ -167,7 +167,12 @@ func AddDeployKey(repoID int64, name, content string, readOnly bool) (*DeployKey
 }
 
 // GetDeployKeyByID returns deploy key by given ID.
-func GetDeployKeyByID(ctx context.Context, id int64) (*DeployKey, error) {
+func GetDeployKeyByID(id int64) (*DeployKey, error) {
```

Comment:
we dont need that helper func just use the context you got

Reply:
Updated. Now there is only one `GetDeployKeyByID(ctx, id)`